packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertListTable.test.tsx

```diff
@@ -117,7 +118,7 @@ describe('Alert List Table test', () => {
 
     await userEvent.click(getByRole('button', { name: 'Enable' }));
 
-    expect(getByText(UPDATE_ALERT_SUCCESS_MESSAGE)).toBeInTheDocument(); // validate whether snackbar is displayed properly
+    expect(getByText(ENABLE_ALERT_SUCCESS_MESSAGE)).toBeInTheDocument(); // validate whether snackbar is displayed properly
```

Comment:
Nit: Better to use `toBeVisible()` than `toBeInTheDocument()`

```suggestion
    expect(getByText(ENABLE_ALERT_SUCCESS_MESSAGE)).toBeVisible(); // validate whether snackbar is displayed properly
```